pages/3.0/_index.md

```diff
@@ -9,17 +9,17 @@ into a Jakarta Servlet.
 
 * [Jakarta Server Pages 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.jsp/releases/3.0.0)
```

Comment:
The release date in this release record will need to be adjusted.  Maybe push for late October.  An Eclipse release review will also need to be initiated at your earliest convenience.  An IP log generated with notes to EMO and PMC.  Should be pretty boilerplate.

Reply:
Changed release date to Oct 26, 2020 for now.

Reply:
The link must be "https://projects.eclipse.org/projects/ee4j.jsp/releases/3.0.0/plan" but not "https://projects.eclipse.org/projects/ee4j.jsp/releases/3.0.0".

Reply:
Either link is fine for the release record.